src/modules/facility-loan-transaction/facility-loan-transaction.service.ts

```diff
@@ -56,12 +56,12 @@ export class FacilityLoanTransactionService {
       amount: loan.LoanAmount,
       issueDate: this.dateStringTransformations.removeTime(loan.EffectiveDate),
       expiryDate: this.dateStringTransformations.removeTime(loan.MaturityDate),
-      spreadRate: pacAccrual.SpreadRate,
+      spreadRate: pacAccrual?.SpreadRate,
```

Comment:
@avaitonis in cases like this we'll end up returning undefined for this field (i.e., missing it out of the response completely) if pacAccrual doesn't exist, whereas the Mulesoft API returns null for this field. Do we care about this discrepancy for GET endpoints or not?

Reply:
Can we please return `null` instead of `undefined` for response robustness.
`null` for a calling application will be suitable.